tools/fp20compiler/rc1.0_general.h

```diff
@@ -99,7 +99,7 @@ class GeneralCombinersStruct {
     void Invoke();
     GeneralCombinerStruct general[RCP_NUM_GENERAL_COMBINERS];
     int num;
-    int localConsts;
+    int localConsts[2];

```

Comment:
This is non descriptive. It could easily be confused with other code for `cc[2]` which has 2 unordered elements.
However, this array is always meant for 2 elements (`[0]` = const0 / `[1]` = const1).
All use is hard-coded to 2 elements, too. So this could theoretically be 2 separate variables like `localConst0Count` / `localConst0Count` or something. Maybe even just `bool hasLocalConst0`.